app/controllers/amusement_parks_controller.rb

```diff
@@ -0,0 +1,7 @@
+class AmusementParksController < ApplicationController
+
+  def show
+    @amusement_park = AmusementPark.find(params[:id])
+    @mechanics = @amusement_park.mechanics.distinct
```

Comment:
We don't want this to live here in our controller. We want this to live in the `amusement_park` model as a method. We can then call that method in our view.